### PR TITLE
Typo: Add a missing installation step in the vagrant setup.

### PR DESCRIPTION
**Pull request by devyanikota**
While using a vagrant virtual machine to setup cockpit, on trying to bring the machine up, it complains about a missing /dist subdirectory.
On running, 
    $ ./autogen.sh
the directory is created preparing the sources for compilation.

Please review the same. Thanks.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -56,13 +56,18 @@ the Cockpit integration tests locally.
...
 
 To use Vagrant to develop Cockpit, run in its top level git checkout.
 In some cases you may need to use `sudo` with vagrant commands:
 
     $ vagrant up
 
+It might complain for a missing `/dist` directory in the Cockpit sources.
+Run it to automatically prepare the sources for compilation.
+
+    $ ./autogen.sh
```
Comment (stefwalter) [inline HACKING.md]: I would suggest just a ```mkdir -p``` would be sufficient here. Or should we just include the dist directory in the tree? Running autogen.sh means installing a lot of extra dependencies.

Comment (devyanikota) [inline HACKING.md]: Ack. testing the suggestions with the edited vagrant file.
Thank you.

**Comment by larskarlitski** [comment]
Thank you! I've merged it manually to remove the first commit and its revert.

**Comment by devyanikota** [comment]
@larskarlitski cool, thank you.

**Pull request closed without merging**